### PR TITLE
Terms update (18/08/2020)

### PR DESCRIPTION
**Pull request by CWKevo**
• Modified the appearance of the page and added a few rules. 
• Most of it stays the same but it must be reviewed.

### CODE REVIEW

**Comment by AndreiJirohHaliliDev2006** [comment]
Just a question: Once this PR is merged, how many days should the newly updated ToS get fully enforced? Do you notify instance admins through email?

File: docs/legal/terms.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+
+**1.** Your forum must not contain spam, porn, or illegal stuff, such as, but not limited to: multi-level marketing schemes, sharing personal details of other people, abuse, hate speech, racism etc... Always check your local laws and regulations before you continue to use this service.
+
+  - FreeFlarum forums are also forbidden to link to external third-party websites & services containing previously mentioned forbidden content.
+
+**2.** FreeFlarum is directed to people who are at least 13 years old or older.
+  - If this server is in the USA, and you are under the age of 13, per the requirements of COPPA (Children's Online Privacy Protection Act), you can't use the FreeFlarum service.
+
+**2.** The service applies a fair use policy towards resources (storage, server capacity and traffic) of free accounts.
+  - The limits are five times the average of all users of the service. Specifically, this means that no more than 250 users per forum are supported.
```
Comment (AndreiJirohHaliliDev2006) [inline docs/legal/terms.md:15]: How the average of the service's users gets calculated?

Comment (CWKevo) [reply docs/legal/terms.md:15]: There is no hard limit, and not any specific calculations. FreeFlarum also runs forums with 250+ users with little no problem, but when you have over 250 users and something starts to break, we can't really help you with that. In average, I'd say that there are at around 40-50 users, so 50×5=250, but I can't really measure the real numbers.

**Comment by CWKevo** [comment]
> Just a question: Once this PR is merged, how many days should the newly updated ToS get fully enforced? Do you notify instance admins through email?

Depends on @gwillem, but I'd post a new post in the Discuss FreeFlarum thread and the new ToS would be effective after 14 days from the date this PR was merged. Not much got changed anyways, just minor adjustments.

**Review by gwillem — APPROVED** [review]
Good job, let's allow 14 days after public notification

**Pull request merged**